### ISSUE TITLE
phyluce_assembly_get_trinity_coverage_for_uce_loci issue

### ISSUE THREAD

**Issue by michael-branstetter** [faircloth-lab/phyluce#142]
While running get_trinity_coverage_for_uce_loci, the script failed at the parsing step. It seems the problem is in picard.py and the get_percent_reads_on_target function which is parsing the metrics file. The metrics file has a new output due to an upgrade in picard and I think that is causing the parsing error.

To try and fix the problem, I commented out lines 181-184 in picard.py and it seems to have worked. This was using phyluce v1.6.7 and the latest version of picard in the bioconda repository.

**Comment by zhangpizhu** [comment]
Hello Dr Brant
I got a problem when run the script, ‘phyluce_assembly_get_trinity_coverage_for_uce_loci’.
This is the revel of terminal.
(phyluce) spring@spring-G41M-ES2L[coverage] phyluce_assembly_get_trinity_coverage_for_uce_loci --assemblies '/home/spring/zpz/coverage/trinity-assemblies' --match-count-output '/home/spring/zpz/coverage/all-taxa-incomplete.conf' --locus-db '/home/spring/zpz/coverage/uce-search-results/probe.matches.sqlite' --output uce_coverage --type untrimmed
NOTE: As we move phyluce to bioconda, some programs are going to
break. The coverage computation scripts are one of those programs
because of their dependence on Picard and GATK.  We are working on a
fix to the problem, which likely involves moving away from Picard and
GATK.  In the meantime, you may need to download your own version of
GATK-lite and put the path to that in your ~/.phyluce.conf

2019-05-14 23:46:58,099 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - == Starting phyluce_assembly_get_trinity_coverage_for_uce_loci ==
2019-05-14 23:46:58,100 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Version: 1.6.6
2019-05-14 23:46:58,100 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --assemblies: /home/spring/zpz/coverage/trinity-assemblies
2019-05-14 23:46:58,100 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --exclude: None
2019-05-14 23:46:58,100 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --locus_db: /home/spring/zpz/coverage/uce-search-results/probe.matches.sqlite
2019-05-14 23:46:58,100 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --log_path: None
2019-05-14 23:46:58,100 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --match_count_output: /home/spring/zpz/coverage/all-taxa-incomplete.conf
2019-05-14 23:46:58,101 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --output: /home/spring/zpz/coverage/uce_coverage
2019-05-14 23:46:58,101 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --resume: None
2019-05-14 23:46:58,101 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --type: untrimmed
2019-05-14 23:46:58,101 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Argument --verbosity: INFO
2019-05-14 23:46:58,101 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Creating the output directory
2019-05-14 23:46:58,101 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Fetching input filenames
2019-05-14 23:46:58,102 - phyluce_assembly_get_trinity_coverage_for_uce_loci - INFO - Fetching loci from all-taxa-incomplete.conf
Traceback (most recent call last):
  File "/home/spring/anaconda2/envs/phyluce/bin/phyluce_assembly_get_trinity_coverage_for_uce_loci", line 331, in <module>
    main()
  File "/home/spring/anaconda2/envs/phyluce/bin/phyluce_assembly_get_trinity_coverage_for_uce_loci", line 304, in main
    raise IOError("There appears to be more than one BAM file for {}".format(organism))
IOError: There appears to be more than one BAM file for sl337892_trinity

I need to download another version of GATK-lite? It’s something wrong with my ‘trinity-assemblies’ files?

**Comment by BenjaminBlanchard** [comment]
I am having the same issue mentioned by @zhangpizhu - "There appears to be more than one BAM file for p_debilis_SRX1164839_trinity". 

There is only one .bam file in the p_debilis_SRX1164839_trinity directory in my designated assemblies file (confirmed manually and with `ls -a *.bam`). And I've already installed/registered GATK (3.8), and it worked fine for the `phyluce_assembly_get_trinity_coverage` function so I don't believe that's the issue?

The function I called is:

phyluce_assembly_get_trinity_coverage_loci \
	--assemblies ./trinity-assemblies \
	--match-count-output ./taxon-sets/all/all-taxa-incomplete_176taxa.conf \
	--type trimmed \
	--locus-db ./uce-search-results_196taxa/probe.matches.sqlite \
	--output UCE_Coverage_176taxa \
	--log-path UCE_Coverage_176taxa_log

I've attached a screenshot of the output.
![Screen Shot 2020-02-08 at 1 52 48 PM](https://user-images.githubusercontent.com/13558463/74091198-a00ea000-4a7a-11ea-9d57-1fe151326be4.png)
